erpnext/crm/custom/contract.json

```diff
@@ -0,0 +1,891 @@
+{
+ "custom_fields": [
```

Comment:
Don't make these custom fields. The doctype lives inside ERPNext itself.

Reply:
Yes I totally agree that these fields belong to doctypes that are present in ERPNext, but atm, it'll too time consuming to move every field individually, will take that up later

Reply:
I think we can take this one a bit slow and sure. Unless we are rushing to get this one out?